sycamore/tests/unit/transforms/test_augment_text.py

```diff
@@ -0,0 +1,42 @@
+import pathlib
+import textwrap
+from sycamore.data import Document
+from sycamore.transforms.augment_text import UDFTextAugmentor, JinjaTextAugmentor
+
+
+class TestAugmentText:
+    doc = Document(
+        {
+            "doc_id": "doc_id",
+            "type": "pdf",
+            "text_representation": "text",
+            "properties": {"path": "/docs/foo.txt", "title": "bar"},
+        }
+    )
+
+    def test_udf_augmentation(self, mocker):
```

Comment:
super minor: You don't need the mocker here, do you? There isn't any mocking happening?

Reply:
nope!